aws-datastore/src/main/java/com/amplifyframework/datastore/storage/sqlite/SQLiteCommandFactory.java

```diff
@@ -220,8 +220,13 @@ public SqlCommand queryFor(@NonNull ModelSchema modelSchema,
             Iterator<QuerySortBy> sortByIterator = sortByList.iterator();
             while (sortByIterator.hasNext()) {
                 final QuerySortBy sortBy = sortByIterator.next();
-                SQLiteColumn column = table.getColumn(sortBy.getField());
-                rawQuery.append(Wrap.inBackticks(column.getAliasedName()))
+                String modelName = Wrap.inBackticks(sortBy.getModelName());
```

Comment:
Will this work if a custom is using models generated the old way?   Don't we need to default this to the current modelName being queried, if `sortBy.getModelName()` is null?

Reply:
Yep, there is a null-check on `modelName` right after this logic to default to the current model right after this :)

Reply:
Oh!  Not sure why I missed that the first time!  Looks good.